src/Console/InstallsInertiaStacks.php

```diff
@@ -113,14 +111,13 @@ protected function installInertiaVueSsrStack()
     {
         $this->updateNodePackages(function ($packages) {
             return [
-                '@inertiajs/server' => '^0.1.0',
                 '@vue/server-renderer' => '^3.2.31',
             ] + $packages;
         });
 
         copy(__DIR__.'/../../stubs/inertia-vue/resources/js/ssr.js', resource_path('js/ssr.js'));
         $this->replaceInFile("input: 'resources/js/app.js',", "input: 'resources/js/app.js',".PHP_EOL."            ssr: 'resources/js/ssr.js',", base_path('vite.config.js'));
-        $this->replaceInFile('});', '    ssr: {'.PHP_EOL."        noExternal: ['@inertiajs/server'],".PHP_EOL.'    },'.PHP_EOL.'});', base_path('vite.config.js'));
+        $this->replaceInFile('});', '    ssr: {'.PHP_EOL."        noExternal: ['@inertiajs/vue3/server'],".PHP_EOL.'    },'.PHP_EOL.'});', base_path('vite.config.js'));
```

Comment:
This is definitely not correct as it doesn't do anything.

When setting it to `@inertiajs/vue` it still externalizes the server itself, so it may need to be `['@inertiajs/vue', '@inertiajs/core']` instead.

Reply:
For what it's worth, I didn't use `noExternal` for any of the new Inertia playgrounds, so I don't think it's required.